samples/perf_client/perf_client.h

```diff
@@ -289,6 +297,10 @@ namespace client
     timing::ResponseTimes response_times;
     timing::CommitPoint last_response_commit = {0, 0};
 
+    std::chrono::high_resolution_clock::time_point last_write_time;
+    int64_t time_elapsed_between_writes_us;
+    int64_t write_delay_us = 0;
```

Comment:
This could be more strongly typed: `std::chrono::nanoseconds write_delay = 0;`

(I'd prefer nanoseconds rather than microseconds so we keep some precision as we approach 100k tx/s)